docs/scim/v1/scimv1.md

```diff
@@ -0,0 +1,361 @@
+We recommend using a dedicated application for SCIM operations.  This way, the customer can differentiate between API and SCIM operations when going through the logs.
+
+##Discovery
+You can use the following endpoints to discover the implementation details of BI's SCIM server.
+
+| Endpoint                                | SCIM spec
+| :---:                                   | :---:         
+| {{baseUrl}}/scim/ServiceProviderConfig  | https://tools.ietf.org/html/rfc7643#section-5
+| {{baseUrl}}/scim/ResourceTypes          | https://tools.ietf.org/html/rfc7643#section-6
+| {{baseUrl}}/scim/Schemas                | https://tools.ietf.org/html/rfc7643#section-7
+
+Where **base_url** is `https://api-us.beyondidentity.com/v1/tenants/{{tenant_id}}/realms/{{realm_id}}/scim/v2`.
+
+##Operations
+The following operations are currently supported by BI SCIM server:
+
+- Create: POST {{baseUrl}}/{{resource}}
+- Read: GET {{baseUrl}}/{{resource}}/{{id}}
+- Replace: PUT {{baseUrl}}/{{resource}}/{{id}}
+- Delete: DELETE {{baseUrl}}/{{resource}}/{{id}}
+- Search: GET {{baseUrl}}/{{resource}}?ﬁlter={{attribute}}{{op}}{{value}}
+
+Where **base_url** is `https://api-us.beyondidentity.com/v1/tenants/{{tenant_id}}/realms/{{realm_id}}/scim/v2`. **resource** can either be `Groups`, `Users`. 
+**id** represents the BI's unique identifier of the corresponding resource.  
+
+###Users
+
+We support the following **Users**`(urn:ietf:params:scim:schemas:core:2.0:User)`attributes:
+
+| User Attribute | Attribute Type| Data Type|Required?|Mutability|Description
+| :---:          | :---:         | :---:    | :---:   | :---:    | :---:     
+| id             | SVA           | string   |N        |N         | ID is unique attribute that identifies the user within a realm.
+| userName       | SVA           | string   |Y        |Y         | The username of the user. The value of this field will be returned as the subject of a OIDC ID Token.
+| displayName    | SVA           | string   |Y        |Y         | The name of the User, suitable for display to end-users.  The name SHOULD be the full name of the User being described, if known.
+| emails         | CMVA          | n/a      |N        |Y         | Email for the User. Only one email is allowed. Primary must be set to true.
+| email.value    | SVA           | string   |Y        |Y         | Address of the user email.
+| email.primary  | SVA           | bool     |Y        |N         | Indicates if this email is primary email of the user.  Currently always set to `true`
+
+SVA = single value attribute
+CMVA = complex multi value attribute
+
+If unsupported attributes are specified in the request, they will be ignored.
+
+**Create user example**
+* Notes:
```

Comment:
I may be wrong, but I'm assuming this is some sort of tip to the developer reading this document. Use the `tip` box:

:::tip Sample App
Sample apps are available to explore. Check out [Example](https://github.com/gobeyondidentity/bi-sdk-js/tree/main/examples/app/js) for the Embedded SDK.
:::

It looks like this:

<img width="995" alt="image" src="https://user-images.githubusercontent.com/6578679/200722334-aea50ad3-6093-4982-8ed1-11e68de60ab8.png">

Reply:
does not look like this syntax works: https://github.com/gobeyondidentity/developer-docs/blob/main/docs/sdks/javascript-sdk/integrate-with-auth0.md

Reply:
oh interesting. Try breaking it up in separate lines like this: https://docusaurus.io/docs/next/markdown-features/admonitions

Reply:
Oh, it's fine if it doesn't work on github. As long as it renders correctly in docusaurus. Is it working in docusaurus when you preview the website?